### ISSUE TITLE
Per Device Default Presets

### ISSUE THREAD

**Issue by abanksss** [antero111/plugin-presets#76]
I am not sure if this would be possible with how runelite works but, it would be nice to have an option to automatically load a certain preset when the client is launched based on the device its running on. With this I could have a default preset for my laptop and desktop with graphics settings for each without having to manually switch after the client is launched. This would allow me to be on the same runelite profile and have all the settings I want synced but still be able to keep my graphics settings separate by default.

**Comment by antero111** [comment]
Can't you have local preset for your graphics settings on each of your devices that you can enable once you switch to a new device? You could have own presets for PC and Laptop to store for example your GPU plugins settings. It's not automatic but really just a single keybind to toggle the preset.

I'm affraid manual toggling would be relative difficult and fairly marginal thing to implement. Problem is to detect the device and whether it is different than last time the RuneLite client was opened on the same profile. I am not sure if RuneLite client has access to such info of the running device.

**Comment by abanksss** [comment]
Yeah I have been switching with local presets on each device which is already much better than having to change everything manually like I used too. Just thought a default would be nice if it is possible.

**Comment by Matermine** [comment]
Would a good workaround be to keep the preset local and set the preset to "load when focused"? You'd have to have one local preset on your PC and a different local preset on your laptop, but it should automatically change your settings when you focus Runelite, a.k.a pretty much automatic.

Let me know if I'm misunderstanding the plugin settings, but it seems like it works.
